src/background.js

```diff
@@ -18,6 +18,16 @@ chrome.runtime.onMessageExternal.addListener(function(request, sender) {
         return;
     }
 
+    // hydrate request
+    if (!request.hasOwnProperty("version")) {
+        request.version = "3.0.12";
+        request.action = "noop";
+    }
```

Comment:
just an idea, it would be interesting to come up with some policy for cleaning up code that was added for backwards compatibility, like after 2 new minor versions or something...

we could then mark such lines with a certain to-do comment, for example

// Remove in: 3.2

what do you think?

Reply:
I like the idea :-).

Reply:
Not that it's really applicable here though, because this is referring to the browserpass version, not the -otp version.

Reply:
We'd still want to remove this block after everyone has updated to any >= 3.0.13 version, right?

Reply:
Yep.